src/data/security-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const securityQuiz = [
+  {
+    Question:
+      "What is the most common way to get hacked?",
+    Answer: "MitM attacks (Man in the Middle Attacks),
```

Comment:
You have a syntax error here because it is missing quotes at the end.
Please fix that before we can move forward with reviewing your question.